### PR TITLE
Feat(aas list) new frontend list

### PR DESCRIPTION
**Pull request by GailMelanie**
# Description

First MVP of the new client side list with a dropdown to select the aas repository.

### CODE REVIEW

File: src/app/[locale]/list/_components-deprecated/AasList.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
+﻿import {
```
Comment (GailMelanie) [inline src/app/[locale]/list/_components-deprecated/AasList.tsx:1]: you can ignore all files in "_components-deprecated" as I didn't change anything (those files will be removed after the new list is finished)

File: src/app/[locale]/list/_components/AasListDataWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    const [selectedRepository, setSelectedRepository] = useState<string | undefined>();
+
+    useAsyncEffect(async () => {
+        if (!selectedRepository) return;
+
+        try {
+            setIsLoadingList(true);
+            const list = await getAasListEntities(selectedRepository!, 10);
+            setAasList(list);
+            setAasListFiltered(list.entities);
```
[the remark this answers is not in the document]
Comment (GailMelanie) [reply src/app/[locale]/list/_components/AasListDataWrapper.tsx:31]: This is the ProductClass filter, I tried to keep it

File: src/app/[locale]/list/_components/AasList.tsx [hunk 3]
```diff
@@ -9,25 +9,38 @@
...
 type AasListProps = {
-    shells: AasListEntry[];
-    tableHeaders: { label: string }[];
+    shells: AasListDto | undefined;
     comparisonFeatureFlag?: boolean;
     selectedAasList: string[] | undefined;
     updateSelectedAasList: (isChecked: boolean, aasId: string | undefined) => void;
 };
 
 export default function AasList(props: AasListProps) {
```
Comment (JonathanXITASO) [inline src/app/[locale]/list/_components/AasList.tsx:25]: you can inline the arguments
```suggestion
export default function AasList({ shells, selectedAasList, updateSelectedAasList, comparisonFeatureFlag }: AasListProps) {
```

File: src/components/basics/StyledImageWithFallBack.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ﻿import { Box, Skeleton, styled } from '@mui/material';
 import { useState } from 'react';
-import DefaultThumbnail from 'assets/AasDefaultThumbnail.svg';
+import DefaultThumbnail from '../../assets/AasDefaultThumbnail.svg';
```
Comment (JonathanXITASO) [inline src/components/basics/StyledImageWithFallBack.tsx:3]: path was fine imo

**Review by JonathanXITASO — APPROVED** [review]

File: src/components/basics/StyledImageWithFallBack.tsx (hunk 4 above)
Comment (GailMelanie) [reply src/components/basics/StyledImageWithFallBack.tsx:3]: will adapt this in the follow up PR

**Pull request merged**